packages/cli/src/lib/index.js

```diff
@@ -120,6 +122,23 @@ export const getPaths = () => {
   }
 }
 
+/**
+ * Format a given string using the config provided in `prettier.config.js`
+ * of a Redwood project.
+ *
+ * @param {string} content - generated content.
+ */
+export const formatTemplate = (content) => {
+  try {
+    const { base: BASE_DIR } = getPaths()
+    const prettierConfig = require(path.join(BASE_DIR, 'prettier.config.js'))
+    return format(content, prettierConfig)
+  } catch (e) {
+    console.log(c.error(e.message))
+    process.exit(0)
```

Comment:
When `prettier.config.js` isn't present, this leaves the generation incomplete. Do we need to just return the provided code if some error pops up?

Reply:
It looks like `options` is optional in `format`, and prettier would fall back to defaults, maybe you could fail silently?